lang_qc/endpoints/pacbio_well.py

```diff
@@ -28,7 +28,8 @@
 
 from lang_qc.db.mlwh_connection import get_mlwh_db
 from lang_qc.db.qc_connection import get_qc_db
-from lang_qc.db.qc_schema import QcState, QcStateHist
+from lang_qc.db.qc_schema import QcState as QcStateDB
```

Comment:
FYI, it also acceptable to do `from lang_qc.db.qc_schema import QcState as QcStateDB, QcStateHist as QcStateDBHist`, perhaps a bit too long for this case

Reply:
This is enforced by isort